enterprise/server/cmd/ci_runner/main.go

```diff
@@ -423,7 +433,26 @@ type actionRunner struct {
 	progressCount int32
 }
 
+func runDebugScript(out io.Writer, script string) {
```

Comment:
script is a slightly odd name for this since you use it to run commands?

Reply:
Renamed to runDebugCommand